showcase/app/templates/layouts/app-frame/index.hbs

```diff
@@ -353,39 +353,29 @@
   </Shw::Grid> --}}
 </section>
 
-{{! For some reason, Ember tests don't play well with iframes (URL not found) so we don't snapshots them in Percy }}
+{{! For some reason, Ember tests don't play well with iframes (URL not found) so we don't have snapshots them in Percy }}
```

Comment:
[todo] I think think "have" works here. if anything we can change to "we can't shapshots them"